gradle/scripts/yaml.gradle

```diff
@@ -13,10 +16,26 @@ buildscript {
         classpath group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.8.8.1'
         classpath group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: '2.8.8'
         classpath group: 'com.github.fge', name: 'json-schema-validator', version: '2.2.6'
+        classpath group: 'org.apache.httpcomponents', name: 'httpcore', version: '4.4.6'
+        classpath group: 'org.asynchttpclient', name: 'async-http-client', version: '2.0.32'
         classpath group: 'org.yaml', name: 'snakeyaml', version: '1.18'
     }
 }
 
+def poll(Collection collection, Closure closure) {
```

Comment:
This method is an attempt to mimic `BlockingQueue#poll()` for a `Collection` using a predicate (`closure`) instead of a timeout.  It's somewhat of a hack as it really does poll rather than wait on some kind of synchronization primitive (e.g. `CountDownLatch`).  The extra effort to do that seemed unnecessary in this case.